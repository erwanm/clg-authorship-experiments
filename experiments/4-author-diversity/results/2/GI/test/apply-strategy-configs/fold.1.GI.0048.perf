0.374448	0.645600	0.580000
