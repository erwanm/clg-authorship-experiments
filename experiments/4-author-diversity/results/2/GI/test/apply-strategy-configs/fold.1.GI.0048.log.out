apply-config.sh: applying config '/tmp/expe4/2/process/selected-meta-configs/fold.1.strategy-as-meta.GI.0048.model/strategy-configs/fold.1.GI.0048' to cases '/home/moreaue/dump/expe4//results/2/GI/test/apply-strategy-configs/only-cases.tmp.lDZBYR4E'; results written to '/home/moreaue/dump/expe4//results/2/GI/test/apply-strategy-configs/fold.1.GI.0048'
apply-config.sh: done.
